Can't find lodash

https://github.com/marionettejs/backbone.radio/pull/209

I'm getting this issue on the radio PR in travis and when running locally.

https://travis-ci.org/marionettejs/backbone.radio/jobs/57445121

```
module.js:340
    throw err;
          ^
Error: Cannot find module 'lodash'
```

lodash is a devDependency and used in the gulpfile.. any chance that's related?  Trying to figure out what else may be different between the implementation here and in marionette

--- a/ci.sh
+++ b/ci.sh
@@ -26,8 +26,8 @@ then
   # Lodash@2 and Lodash@3 have different file structures
   if [[ "$LODASH" < "3.0" ]]
   then
-    mv node_modules/lodash/lodash.js node_modules/underscore/underscore.js
-  else 
-    mv node_modules/lodash/index.js node_modules/underscore/underscore.js
+    cp node_modules/lodash/lodash.js node_modules/underscore/underscore.js
+  else
+    cp node_modules/lodash/index.js node_modules/underscore/underscore.js
   fi
 fi

